rio-code/src/main/java/frc/robot/Commands/DriveCommands/AlignWithLeftReef.java

```diff
@@ -100,6 +100,9 @@ public void initialize() {
         headingController.setSetpoint(targetReefPose.getRotation().plus(Rotation2d.k180deg).getDegrees());
         headingController.reset();
 
+        xController.setTolerance(0.02);
```

Comment:
No magic numbers please. Add to constants.
Also, are you sure you want to set a different deadband just for this command?